examples/inspiral/check_GW150914_detection.py

```diff
@@ -0,0 +1,37 @@
+# Read a pycbc_inspiral HDF5 trigger file and check that it contains triggers
+# compatible with GW150914
+# 2016 Tito Dal Canton
+
+import sys
+import h5py
+import numpy as np
+
+
+# GW150914 params from my run
+# https://www.atlas.aei.uni-hannover.de/~tito/LSC/er8/er8b_c00_1.2.0_run1
+gw150914_time = 1126259462.4
+gw150914_snr = {'H1': 19.71, 'L1': 13.28}
+gw150914_chi2r = {'H1': 1.05, 'L1': 0.45}
+
+f = h5py.File(sys.argv[1], 'r')
+detector = f.keys()[0]
+end_times = f[detector]['end_time'][:]
+snrs = f[detector]['snr'][:]
+chi2rs = f[detector]['chisq'][:] / (2 * f[detector]['chisq_dof'][:] - 2)
+
+# search for trigs compatible with GW150914
```

Comment:
May need to check with @titodalcanton  who I think informed the original settings. The main point here is to have a simple test rather than nothing.